### DISCUSSION TITLE
New User but looks like i'm in wrong place??

### DISCUSSION THREAD

**Discussion by SteveK50** [community/community#123017] [New to GitHub]
Got here trying to load "Outlook Google Calendar Sync"... 

want to know how to change initial calendar sync, from 30 days, to go back and sync ALL (since 2020) Outlook calendar data with Google.

Only thing closest to "HELP" in OGCS User Guide is this place.

Can anyone else?  Point me to an OGCS Help Area?  
Thanks in advance,
Steve

**Comment by infonatica** [comment]
Same problem here :)
